store/sqlstore/user_store.go

```diff
@@ -228,9 +228,10 @@ func (us SqlUserStore) UpdateUpdateAt(userId string) store.StoreChannel {
 
 		if _, err := us.GetMaster().Exec("UPDATE Users SET UpdateAt = :Time WHERE Id = :UserId", map[string]interface{}{"Time": curTime, "UserId": userId}); err != nil {
 			result.Err = model.NewAppError("SqlUserStore.UpdateUpdateAt", "store.sql_user.update_update.app_error", nil, "user_id="+userId, http.StatusInternalServerError)
-		} else {
-			result.Data = userId
+			return
 		}
+
+		result.Data = curTime
```

Comment:
This is a functional change, but the return value was unused in the two places where this was used:
https://github.com/mattermost/mattermost-server/blob/aba194188fc6f75202c60078fe4ce543fc8c064a/app/team.go#L479-L481
https://github.com/mattermost/mattermost-server/blob/aba194188fc6f75202c60078fe4ce543fc8c064a/app/team.go#L802-L804

Returning the assigned time here makes unit testing this simpler.